app/helpers/facility_reservations_helper.rb

```diff
@@ -1,5 +1,19 @@
 module FacilityReservationsHelper
 
+  def offline_category_collection
+    OfflineReservation::CATEGORIES.map do |category|
+      [reservation_category_label(category), category]
+    end
+  end
+
+  def reservation_category_label(reservation_category)
+    if reservation_category.present?
+      I18n.t("offline_reservations.categories.#{reservation_category}")
```

Comment:
I was hoping we could do something like `t(reservation_category, scope: "offline_reservations.categories", default: "")` to get rid of the conditional, but you still need some value going into the first argument.

Do we need the `I18n.` namespace?